Source/WebCore/CMakeLists.txt

```diff
@@ -2296,13 +2296,20 @@ set(WebCoreTestSupport_INTERFACE_INCLUDE_DIRECTORIES
 )
 
 WEBKIT_FRAMEWORK_DECLARE(WebCore)
-WEBKIT_FRAMEWORK_DECLARE(WebCoreTestSupport)
+WEBKIT_LIBRARY_DECLARE(WebCoreTestSupport ${WebCoreTestSupport_LIBRARY_TYPE})
```

Comment:
Why do we need the second argument `${WebCoreTestSupport_LIBRARY_TYPE}` here?

Reply:
Whoops at an earlier iteration there was a second parameter. Good catch!